app/src/main/java/org/fossasia/openevent/general/search/SearchTimeFragment.kt

```diff
@@ -105,7 +105,7 @@ class SearchTimeFragment : Fragment() {
     }
 
     private fun redirectToSearch(time: String) {
-        val args = SearchFragmentArgs.Builder(time).build().toBundle()
+        val args = SearchFragmentArgs.Builder().setStringSavedDate(time).build().toBundle()
```

Comment:
I made this change because for some reason that I could not find, the constructor for `SearchFragmentArgs.Builder` that takes a string as an argument appears to be missing from the generated file of `SearchFragmentArgs.java`

This is the missing constructor:

`  public Builder(@Nullable String StringSavedDate) {
      this.arguments.put("@string/saved_date", StringSavedDate);
    }
`